lib/std/fmt.zig

```diff
@@ -1381,11 +1420,20 @@ test "slice" {
     {
         var runtime_zero: usize = 0;
         const value = @intToPtr([*]align(1) const []const u8, 0xdeadbeef)[runtime_zero..runtime_zero];
-        try testFmt("slice: []const u8@deadbeef\n", "slice: {}\n", .{value});
+        try testFmt("slice: []const u8@deadbeef\n", "slice: {*}\n", .{value});
     }
 
     try testFmt("buf:  Test\n", "buf: {s:5}\n", .{"Test"});
     try testFmt("buf: Test\n Other text", "buf: {s}\n Other text", .{"Test"});
+
+    {
+        var int_slice = [_]u32{ 1, 4096, 391891, 1111111111 };
+        var runtime_zero: usize = 0;
+        try testFmt("int: [1, 4096, 391891, 1111111111]", "int: {}", .{int_slice[runtime_zero..]});
```

Comment:
Is there a reason why slices don't use curly braces like arrays?

Reply:
Yeah, style-wise there are a few questions. Do we want a space after the opening/closing brace? Do we want curly or square brackets? Same symbol for both arrays and slices?

Reply:
If we want to make generating Zig with the formatting system easier we should definitely use curly braces for both. For the spaces I'd either follow `zig fmt` or just leave it as is to keep things simple.